### PR TITLE
Added support to get the latest public events on github

### PR DESCRIPTION
**Pull request by GeorgeL9**

### CODE REVIEW

File: github/MainClass.py [hunk 1]
```diff
@@ -247,6 +248,24 @@ def get_gists(self):
...
         )
 
+    def get_events(self, since=github.GithubObject.NotSet):
+        """
+        :calls: `GET /events <http://developer.github.com/v3/events>`_
+        :param since: integer
+        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.Event.Event`
+        """
+        assert since is github.GithubObject.NotSet or isinstance(since, (int, long)), since
+        url_parameters = dict()
```
Comment (jzelinskie) [inline github/MainClass.py]: it's more idiomatic to use `{}` instead of `dict()`

**Comment by jzelinskie** [comment]
If you can write a test -- I'll merge.

**Comment by GeorgeL9** [comment]
I'll try to get to it this week.

**Comment by GeorgeL9** [comment]
I'm having some difficulties getting the tests setup. I added the GithubCredentials.py to the /tests folder, but I am seeing a lot of errors:

Ran 400 tests in 1.191s
FAILED (failures=144, errors=150)

Could someone point me in the right direction with this?

**Comment by sfdye** [comment]
@GeorgeL9 Thanks for you PR. I am closing because this API was depreciated by the:
https://developer.github.com/v3/activity/events/

and was already implemented by someone else.

To answer your question, before running the tests you need to record the test replay data. For detailed information you can refer to the docs here:
https://github.com/PyGithub/PyGithub/blob/master/CONTRIBUTING.md#automated-tests

**Pull request closed without merging**